scripts/build.sh

```diff
@@ -1,28 +1,26 @@
 #!/usr/bin/env bash
+set -e
 
-if [ -z "$1" ]; then
+ROOTDIR="$(dirname "$0")/.."
+BUILDDIR="$(mktemp --tmpdir="${ROOTDIR}" --directory build.XXXXXXXX)"
+
+if [[ $# -eq 0 ]]; then
```

Comment:
Could someone verify that this has to be 0 instead of 1?

Reply:
Yes that's correct